activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -173,10 +173,13 @@ def initialize(config_or_deprecated_connection, deprecated_logger = nil, depreca
       end
 
       def synchronized=(synchronized) # :nodoc:
-        @lock = if synchronized
-          ActiveSupport::Concurrency::LoadInterlockAwareMonitor.new
-        else
-          ActiveSupport::Concurrency::NullLock
+        @lock = case synchronized
```

Comment:
Newline after the `=` will convince rubocop to allow formatting this in a way that looks less terrible